jbpm-services/jbpm-services-ejb/jbpm-services-ejb-timer/src/main/java/org/jbpm/services/ejb/timer/EJBTimerScheduler.java

```diff
@@ -230,7 +223,11 @@ private boolean isSessionNotFound(Exception e) {
             operation.doWork(item);
             utx.commit();
         } catch(RollbackException e) {
-            logger.warn("Transaction was rolled back for {}", item);
+            logger.warn("Transaction was rolled back for {} with status {}", item, utx.getStatus());
+            if(utx.getStatus() == javax.transaction.Status.STATUS_ACTIVE) {
```

Comment:
One question jere: should other states be considered here as STATUS_MARKED_ROLLBACK or only applies to active?